src/keyring/KeyringController.test.ts

```diff
@@ -104,11 +106,13 @@ describe('KeyringController', () => {
     expect(identitiesLength).toStrictEqual(initialIdentitiesLength);
   });
 
-  it('should create new vault and keychain', async () => {
+  it('should create new vault, mnemonic and keychain', async () => {
     const currentState = await keyringController.createNewVaultAndKeychain(
       password,
     );
     expect(initialState).not.toBe(currentState);
+    const currentSeedWord = await keyringController.exportSeedPhrase(password);
+    expect(currentSeedWord).toBeDefined();
```

Comment:
Is there any way for us to stub whatever is responsible for generating the seed phrase so we can be more exact with this assertion?

Reply:
This would be stubbing `generateRandomMnemonic` on  `HDKeyring` from `@metamask/eth-hd-keyring` just fyi @jpuri

Reply:
`@metamask/eth-hd-keyring` is a transitive dependency and I found it hard to mock it.